mmsoap/client.py

```diff
@@ -1,14 +1,14 @@
 """
```

Comment:
Not sure what you're convention is but I'm familiar with the block comment specified by the PEP guideline:
https://www.python.org/dev/peps/pep-0008/#block-comments
Which specifies each line with a #